incidents/hortense_20251016.yml

```diff
@@ -0,0 +1,19 @@
+meta_data:
+  title: Security updates of GPU nodes on Tier-1 Hortense
```

Comment:
The agreement was made last time that we don't mention the word security in such an announcement. @epauwels ?